app/fight.py

```diff
@@ -0,0 +1,24 @@
+from app.knights import Knight
+
+
+class Arena:
+
+    @staticmethod
+    def battle(knights: list[Knight]) -> None:
+        knight1 = Knight
+        knight2 = Knight
+        knight3 = Knight
+        knight4 = Knight
```

Comment:
This part kind of mess me. Don’t create objects of class without parameters. Also, it would be better to name variables by knight’s name.